### ISSUE TITLE
name?

### ISSUE THREAD

**Issue by timelyportfolio** [timelyportfolio/g2r#3]
I chose `g2r` to start the project, but I am by no means wedded to the name?  Are there other suggestions?

**Comment by baptiste** [comment]
grr

**Comment by mdequeljoe** [comment]
@timelyportfolio what about all caps - G2R (or RG2)?

**Comment by juba** [comment]
I tend to dislike uppercase in package names, so `g2r` seems nice. But the source package is named `G2`, so do we have to keep the `G` in the R package name ?

**Comment by timelyportfolio** [comment]
I agree that I would like to stick to lowercase for easier typing.  In the past, I added a `R` but there is no telling how much time I have lost to `<shift>+R` :).
